commands/mrHelper.js

```diff
@@ -538,18 +540,30 @@ module.exports = {
             { name: 'alphabetical', value: 'alphabetical' },
             { name: 'level', value: 'level' },
         ))
-    ),
+    )
+    .addSubcommand((subcommand) => subcommand.setName('says').setDescription('Hear what the Hero of Orgrimmar has to say'))
```

Comment:
I think we should have some text that clarifies this will be private...

Reply:
Updated this description to "The Hero of Orgrimmar will whisper sweet nothings into your ear" to try and convey that it will only be visible to the user. Do you think this is any better?

Reply:
LOL yeah that's fine we can update it later if needed